### PR TITLE
[improve](function) get result from constant_col when expr is const

### PR DESCRIPTION
**Pull request by zhangstar333**
## Proposed changes
const expr have execute in open function.
so in execute function could get result directly.

Issue Number: close #xxx

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by zhangstar333** [comment]
run buildall

**Comment by zhangstar333** [comment]
run buildall

**Comment by zhangstar333** [comment]
run buildall

File: be/src/vec/exprs/vin_predicate.cpp [hunk 1]
```diff
@@ -94,6 +95,12 @@ void VInPredicate::close(VExprContext* context, FunctionContext::FunctionStateSc
 }
 
 Status VInPredicate::execute(VExprContext* context, Block* block, int* result_column_id) {
+    if ((_constant_col != nullptr) && is_constant()) { // const have execute in open function
```
Comment (HappenLee) [inline be/src/vec/exprs/vin_predicate.cpp:98]: use a function to do the same work

**Comment by zhangstar333** [comment]
run buildall

File: be/src/vec/exprs/vexpr.cpp [hunk 2]
```diff
@@ -554,4 +554,12 @@ Status VExpr::check_constant(const Block& block, ColumnNumbers arguments) const
     return Status::OK();
 }
 
+Status VExpr::get_result_from_const(vectorized::Block* block, const std::string& expr_name,
```
Comment (HappenLee) [inline be/src/vec/exprs/vexpr.cpp:557]: void do nothing

**Review by HappenLee — APPROVED** [review]
LGTM

**Review by zclllyybb — APPROVED** [review]

**Pull request merged**